test/data/lib/upload-forms/merge-properties/forms/example.properties.json

```diff
@@ -5,7 +5,7 @@
     "place": false
   },
   "xml2sms": "hello world",
-  "internalId": "different",
+  "internalId": "example",
```

Comment:
I updated this test case to have a valid (matching) `internalId` to avoid the new error.  Then I added the `invalid-internal-id` case to test the error.